concent_api/concent_api/middleware.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, get_response: Callable) -> None:
 
     def __call__(self, request: HttpRequest) -> HttpResponse:
         response = self.get_response(request)
-        response['Concent-Golem-Messages-Version'] = __version__
+        response['Concent-Golem-Messages-Version'] = GOLEM_MESSAGES_VERSION
```

Comment:
Please also change this class description to 
`Used to attach version of the golem messages package supported by this Concent instance to HTTP response header.`

And remove inheritance from `object`.